pkg/webhook/admission/webhook.go

```diff
@@ -121,6 +123,8 @@ func (w *Webhook) Handle(ctx context.Context, req atypes.Request) atypes.Respons
 	case types.WebhookTypeValidating:
 		resp = w.handleValidating(ctx, req)
 	default:
+		metrics.FailedRequests.WithLabelValues(w.Name).Inc()
```

Comment:
I have a general question. Maybe I missed that part. It looks that the failure branches will have metrics. How about the successful ones?

Reply:
This is a WIP PR.
I created this PR to see if Travis can reproduce the same strange error I have locally.
I will push more change for the successful ones.